debian/rules.in

```diff
@@ -31,40 +33,41 @@ export TIME=$(shell LANG=C date --date='@$(TIMESTAMP)' '+%T')
 kernel_version = @KERNEL_VERSION@
 configure_realtime_arg = @CONFIGURE_REALTIME_ARG@
 enable_build_documentation = @ENABLE_BUILD_DOCUMENTATION@
+SRCDIR = $(CURDIR)/src
 DESTDIR=$(CURDIR)/debian/tmp
 DEV_PACKAGE_NAME=@MAIN_PACKAGE_NAME@-dev
 
+export PYTHON=/usr/bin/python3
+
 %:
-	dh $@
+	dh $@ -D$(SRCDIR)
 
 override_dh_auto_configure:
 	cd src && ./autogen.sh
-	cd src && PYTHON=/usr/bin/python3 ./configure \
+	dh_auto_configure -- \
 	    --prefix=/usr --sysconfdir=/etc \
+	    --libdir=/usr/lib \
+	    --disable-multiarch \
```

Comment:
Are you sure you want to disable multiarch?

Reply:
I don't want to turn it off.
Changing library paths requires a thorough analysis of the source code that searches for these libraries or
sets this path at the very beginning when starting on a running system.

In addition, there are explicitly prescribed paths for x86:
```bash
linuxcnc> grep -nr 'lib/x86' ./*
....
./lib/python/qtvcp/designer/install_script:85:sopath=/usr/lib/x86_64-linux-gnu/qt5/plugins/designer/
./lib/python/qtvcp/designer/install_script:98:        echo -e '/usr/lib/x86_64-linux-gnu/qt5/plugins/designer/'
./scripts/setup_designer.in:85:sopath=/usr/lib/x86_64-linux-gnu/qt5/plugins/designer/
./scripts/setup_designer.in:98:        echo -e '/usr/lib/x86_64-linux-gnu/qt5/plugins/designer/'
....
linuxcnc> grep -nr '/lib' ./scripts/*
./scripts/linuxcnc.in:79:           LD_LIBRARY_PATH=$LINUXCNC_HOME/lib
./scripts/linuxcnc.in:81:           LD_LIBRARY_PATH=$LINUXCNC_HOME/lib:"$LD_LIBRARY_PATH"
.....
./scripts/runtests.in:28:    export LIBDIR=${TOPDIR}/lib
./scripts/runtests.in:35:    export LIBDIR=@EMC2_HOME@/lib
```
And this needs to be fixed in any case.

We will reverse this commit in the next pull request when I analyze and
test it on running hardware that is different from x86